services/ui-src/src/forms/mcpar/ebssei/ebssei.json

```diff
@@ -1,18 +1,170 @@
 {
   "path": "/mcpar/bss-entity-indicators",
   "intro": {
-    "section": "E: BSS Entity Indicators",
-    "subsection": "X: Program Integrity"
+    "section": "Section E: BSS Entity Indicators",
+    "subsection": "Topic IX. Beneficiary Support System (BSS) Entities",
+    "spreadsheet": {
+      "title": "Find in the Excel Workbook",
+      "descriptionList": ["E_BSS_Entities"]
+    },
+    "info": [
+      {
+        "type": "html",
+        "content": "<p>Per 42 CFR 438.66(e)(2)(ix), the Managed Care Program Annual Report must provide information on and an assessment of the operation of the managed care program including activities and performance of the beneficiary support system. Information on how BSS entities support program-level functions is on the </p>"
+      },
```

Comment:
Wrapping this in `<p>` creates a line break between this and the next element (see screenshot). 

<img width="569" alt="Screen Shot 2022-08-24 at 17 31 57" src="https://user-images.githubusercontent.com/27705928/186535296-3a5904e9-815b-4a18-97df-265eccbc6edc.png">

There are two ways to fix this: 
  1. Change the `<p>` tags to `<span>` tags, or
  2. Remove the `<p>` tags and change the type to `"type": "span"`.